for_developers/release_checklist/index.md

```diff
@@ -102,6 +102,10 @@ This assumes you try to create `v6-22-00-patches`, adjust accordingly.
       - Add a 'highlights' section in the generated release page.
       - If this applies, mark the release as `state: latest` and remove the attribute to the one previously holding it (`git grep "state: latest" -- _releases/`)
       - Create a PR against `root-project/web`.
+  1. Update the list of [build options](https://root.cern/install/build_from_source/#all-build-options)
+      - `cd` into main directory of the `root-project/web` repository.
+      - Run `bash .github/scripts/generateBuildOptions.sh v6-22-00-patches`. This creates the file `_includes/build_options_v6-22-00-patches.md`.
```

Comment:
We should probable have `generateBuildOptions.sh` and `make-release-page.py` (currently at `_releases/`) in the same directory. If you agree, could you move one next to the other?